pkg/synthesis/model/symbolicexpr/groupBasedInternalResource.go

```diff
@@ -103,15 +103,16 @@ func getConjunctionOperator(isExcluded bool, elem collector.ExpressionElement,
 	return &retOp
 }
 
-func getTermForExprElement(config *configuration.Config, isExcluded bool, elem collector.ExpressionElement,
-	group string) []atomic {
+// return  DNF which is a symbolic presentation of the expression element
+func getConjunctionsForExprElement(config *configuration.Config, isExcluded bool, group string,
```

Comment:
rename this function:
`getDNFForExprElement`

Reply:
I would also do some search over code files, to make sure old `conjunction` has not been left in other places.